packages/terra-switch/src/terra-dev-site/doc/example/DefaultSwitch.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+import Switch from 'terra-switch';
+
+export default () => {
+  const [isOn, setValue] = useState(false);
+  return (
+    <div>
+      <Switch
+        isOn={isOn}
+        labelText="Label"
+        onChange={() => { setValue(!isOn); }}
```

Comment:
This is doing a blind setState, this should probably utilize the value returned from the onChange

Reply:
updated to change value based on value received from [Onchange callback()](https://github.com/cerner/terra-core/blob/3ccbde10c12629e3a8a6a617536b9c403ec693c7/packages/terra-switch/src/terra-dev-site/doc/example/SwitchWithOnChange.jsx#L13)